sw/device/silicon_creator/rom/e2e/chip_specific_startup.c

```diff
@@ -110,12 +117,11 @@ status_t test_chip_specific_startup(ujson_t *uj) {
 
   // We need to know the OTP configs for AST init and jitter in order to
   // correctly determine the pass/fail conditions.
-  TRY(dif_otp_ctrl_read_blocking(
-      &otp_ctrl, kDifOtpCtrlPartitionCreatorSwCfg,
-      OTP_CTRL_PARAM_CREATOR_SW_CFG_AST_INIT_EN_OFFSET,
-      &cs.otp.creator_sw_cfg_ast_init_en, 1));
   TRY(dif_otp_ctrl_read_blocking(&otp_ctrl, kDifOtpCtrlPartitionCreatorSwCfg,
-                                 OTP_CTRL_PARAM_CREATOR_SW_CFG_JITTER_EN_OFFSET,
+                                 kAstInitEnOffset,
+                                 &cs.otp.creator_sw_cfg_ast_init_en, 1));
+  TRY(dif_otp_ctrl_read_blocking(&otp_ctrl, kDifOtpCtrlPartitionCreatorSwCfg,
+                                 kJitterEnOffset,
```

Comment:
This interface is non-ideal since our generated constants are relative to the start of the memory mapped window, not individual partitions. We should probably just remove the partition parameter from this function and perform the addition, if needed, at callsites.

Reply:
Created #16394